venia-integration-tests/src/tests/pageBuilder/integrationTests/banner.spec.js

```diff
@@ -0,0 +1,146 @@
+                timeout: 60000
+
+            });
+        });
+    });
+    it('verify banner content2', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', { fixture: 'pageBuilder/banner/banner2.json' }).as('getCMSMockData')
+        cy.visit('/');
+        cy.wait(['@getCMSMockData']).its('response.body')
+        cy.wait(5000);
+        cy.loadFullPage().then(() => {
+            cy.captureFullPageScreenshot({
+                name: 'Page Builder Home Page2',
+                timeout: 60000
+
+            });
+        });
+    });
+
+    it('verify banner content3', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', { fixture: 'pageBuilder/banner/banner3.json' }).as('getCMSMockData')
+        cy.visit('/');
+        cy.wait(['@getCMSMockData']).its('response.body')
+        cy.wait(5000);
+        cy.loadFullPage().then(() => {
+            cy.captureFullPageScreenshot({
+                name: 'Page Builder Home Page3',
+                timeout: 60000
+
+            });
+        });
+    });
+
+    it('verify banner content4', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', { fixture: 'pageBuilder/banner/banner4.json' }).as('getCMSMockData')
+        cy.visit('/');
+        cy.wait(['@getCMSMockData']).its('response.body')
+        cy.wait(5000);
+        cy.loadFullPage().then(() => {
+            cy.captureFullPageScreenshot({
+                name: 'Page Builder Home Page4',
+                timeout: 60000
+
+            });
+        });
+    });
+
+    it('verify banner content5', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', { fixture: 'pageBuilder/banner/banner5.json' }).as('getCMSMockData')
+        cy.visit('/');
+        cy.wait(['@getCMSMockData']).its('response.body')
+        cy.wait(5000);
+        cy.loadFullPage().then(() => {
+            cy.captureFullPageScreenshot({
+                name: 'Page Builder Home Page5',
+                timeout: 60000
+
+            });
+        });
+    });
+
+    it('verify banner content6', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', { fixture: 'pageBuilder/banner/banner6.json' }).as('getCMSMockData')
+        cy.visit('/');
+        cy.wait(['@getCMSMockData']).its('response.body')
+        cy.wait(5000);
+        cy.loadFullPage().then(() => {
+            cy.captureFullPageScreenshot({
+                name: 'Page Builder Home Page6',
+                timeout: 60000
+
+            });
+        });
+    });
+
+    it('verify banner content7', () => {
+        cy.intercept('GET', '**/graphql?query=query+GetCmsPage*', { fixture: 'pageBuilder/banner/banner7.json' }).as('getCMSMockData')
+        cy.visit('/');
+        cy.wait(['@getCMSMockData']).its('response.body')
+        cy.wait(5000);
+        cy.loadFullPage().then(() => {
+            cy.captureFullPageScreenshot({
+                name: 'Page Builder Home Page7',
+                timeout: 60000
```

Comment:
Is there a way to avoid these high timeouts? and rather wait for a piece of content we expect to render near the end of the content?

Reply:
This is max time given to plugin to process screenshot updates/comparison and not related to page load. Also it does not actually wait 60 secs every time, as soon as screen comparison is done control does move to next step.

Reply:
Ah super cool, thanks for clarifying.